### PR TITLE
Feat : Added text-utilities Component

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added a text-utilities Component in Aqua_UI.
Screenshot (299)

### CODE REVIEW

**Review by ajit-sharmaGH — APPROVED** [review]
Looks nice the charming color
Good naming for every class selectors
Good to merge

**Review by Nagashree01 — APPROVED** [review]
code is clean. good to merge

File: Component-assets/Css/ComponentsStyles.css [hunk 1]
```diff
@@ -593,109 +590,190 @@ cursor: pointer;
...
+}
+.stacked-list--items {
+  border: 2px solid var(--secondary-bg);
+  margin: 4px;
+  padding: 4px;
+}
+
+/* Rating */
+.rating {
+  display: flex;
```
Comment (paarthbajaj) [inline Component-assets/Css/ComponentsStyles.css:750]: I found this display flex property to be very repetitive in your code. You could make a utility class out of it.

**Review by paarthbajaj — APPROVED** [review]
I saw that you have added a lot of utility classes but you have to implement them in your own project too. The repetitive code will be a pain for you in the future. So I suggest you, refactor some of your code. Thanks and well done.

**Comment by SaurabhChawda** [comment]
> Looks nice the charming color Good naming for every class selectors Good to merge

Thank You

**Comment by SaurabhChawda** [comment]
> code is clean. good to merge

Okay, Thank You

File: Component-assets/Css/ComponentsStyles.css (hunk 1 above)
Comment (SaurabhChawda) [reply Component-assets/Css/ComponentsStyles.css:750]: okay, I will try to make changes.

**Comment by SaurabhChawda** [comment]
> I saw that you have added a lot of utility classes but you have to implement them in your own project too. The repetitive code will be a pain for you in the future. So I suggest you, refactor some of your code. Thanks and well done.

Okay, I will keep in mind and code according to your suggestion, Thank you for the deep review.

**Pull request merged**